### PR TITLE
Allow to change navigation bar tint color and style

### PR DESCRIPTION
**Pull request by pablog**
## Summary ##

Allow to change navigation bar tint color (arrow and bar buttons color).
Allow to change navigation bar style.

### CODE REVIEW

**Comment by woloxci** [comment]
**CodeStats metrics details:**

:white_check_mark: Code Coverage (Slather): 87.2
:white_check_mark: Code Quality (Swiftlint): 98.0


Click [here](http://codestats.wolox.com.ar/organizations/wolox/projects/wolmo-core-ios/branches/navigation-controller-tint-color) for more details.

**Comment by woloxci** [comment]
**CodeStats metrics details:**

:white_check_mark: Code Coverage (Slather): 87.2
:white_check_mark: Code Quality (Swiftlint): 98.0


Click [here](http://codestats.wolox.com.ar/organizations/wolox/projects/wolmo-core-ios/branches/navigation-controller-tint-color) for more details.

**Comment by pablog** [comment]
@nahuelwolox Done! This is definitely confusing.

**Comment by woloxci** [comment]
**CodeStats metrics details:**

:white_check_mark: Code Coverage (Slather): 87.2
:white_check_mark: Code Quality (Swiftlint): 98.0


Click [here](http://codestats.wolox.com.ar/organizations/wolox/projects/wolmo-core-ios/branches/navigation-controller-tint-color) for more details.

**Comment by woloxci** [comment]
**CodeStats metrics details:**

:white_check_mark: Code Coverage (Slather): 87.2
:white_check_mark: Code Quality (Swiftlint): 98.0


Click [here](http://codestats.wolox.com.ar/organizations/wolox/projects/wolmo-core-ios/branches/navigation-controller-tint-color) for more details.

File: Core/Extensions/UIKit/UIViewController.swift [hunk 1]
```diff
@@ -78,6 +78,30 @@ public extension UIViewController {
     }
     
     /**
+     Configures the navigation bar tint color.
+     - parameter color: The new tint color of the navigation bar. This represents the navigation bar tint color.
```
Comment (danielaRiesgo) [inline Core/Extensions/UIKit/UIViewController.swift]: I think `This represents the navigation bar tint color.` is redundant.

Comment (pablog) [inline Core/Extensions/UIKit/UIViewController.swift]: Yes, I changed that to avoid repeating the same than before.

**Comment by pablog** [comment]
@danielaRiesgo let me know if it's ready to merge with these new comments.

**Comment by woloxci** [comment]
**CodeStats metrics details:**

:white_check_mark: Code Coverage (Slather): 87.2
:white_check_mark: Code Quality (Swiftlint): 98.0


Click [here](http://codestats.wolox.com.ar/organizations/wolox/projects/wolmo-core-ios/branches/navigation-controller-tint-color) for more details.

**Pull request merged**